tests/all/test_passes.rs

```diff
@@ -142,18 +165,20 @@ fn test_pass_manager_builder() {
 
     pass_manager_builder.populate_module_pass_manager(&module_pass_manager);
 
-    let module2 = module.clone();
+    #[cfg(not(feature = "llvm15-0"))]
+    {
+        let module2 = module.clone();
+
+        let lto_pass_manager = PassManager::create(());
+        pass_manager_builder.populate_lto_pass_manager(&lto_pass_manager, false, false);
```

Comment:
Please refresh my memory - was lto pass manager removed in 15?

Reply:
I couldn't find this in the changelogs, but it [was removed from the underlying llvm-sys](https://github.com/tari/llvm-sys.rs/blob/llvm-14.0/src/transforms/pass_manager_builder.rs#L43), so it makes sense, because all the old pass manager stuff gets less and less support.